bin/bcoin-cli

```diff
@@ -127,6 +128,19 @@ class CLI {
     this.log(filter);
   }
 
+  async estimateFee() {
+    const blocks = this.config.uint(0, 1);
+
+    const fee = await this.client.estimateFee(blocks);
+
+    if (!fee || !fee.rate) {
```

Comment:
Intentional to have the fee of `0` log as not found?

Reply:
I thought this is what we decided: https://github.com/bcoin-org/bcoin/pull/889#discussion_r338199173

Also, we certainly don't want users trying to send a tx with `0`-fee. So probably an error message like this is the most useful.

Reply:
"Not found" is different than "Fee found and was zero".

Reply:
I would probably just skip `!fee.rate` in the if here, and log `0` as the fee.